Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup Label="Build">
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
I believe .NET 8 was officially released today. Would it be possible to switch to latest here?

Reply:
I'm not sure if it was a bug in my setup, but in order to use primary constructors I had to set this to `preview` instead of `latest`.

We still target the .NET 6 runtime, but are using the .NET 8 SDK to compile our package. I wonder if it's just an issue with my setup, or if primary constructors in packages targeting .NET 6 require this to be `preview`.

Reply:
It was an issue with my local setup. I tested with `latest` in a fresh codespace and it worked no problem.